libs/ui/elements/src/lib/metadata-info/metadata-info.component.html

```diff
@@ -106,9 +106,23 @@
     </div>
     <div *ngIf="metadata.uniqueIdentifier">
       <p class="text-sm" translate>record.metadata.uniqueId</p>
-      <p class="text-primary font-medium">
-        {{ metadata.uniqueIdentifier }}
-      </p>
+      <div class="flex flex-row content-align gap-2">
+        <button
```

Comment:
Could we reuse the `gn-ui-copy-text-button` here?

Reply:
I will look into doing that :+1: